### PR TITLE
Feature/add grunt to compatibility tests

### PR DESCRIPTION
**Pull request by dwybourn**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,8 +18,9 @@
     "test:smoke": "npm-run-all -s test:smoke:test test:smoke:report",
     "test:smoke:test": "mkdir -p build/reports && cross-env TESTS=smoke cucumber-js src/test/smoke/features/ --require 'src/test/common/steps/*.js' --require 'src/test/smoke/config' -f json:build/reports/smoke-report.json",
     "test:smoke:report": "node src/test/smoke/smoke-report.js",
-    "test:compatibility": "mkdir -p build/reports && cross-env TESTS=compatibility cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config' -f json:build/reports/compatibility-report.json",
-    "test:compatibility:debug": "node $NODE_DEBUG_OPTION node_modules/cucumber/bin/cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config' -f json:build/reports/compatibility-report.json",
+    "test:compatibility": "mkdir -p build/reports && cross-env TESTS=compatibility grunt compatibility",
+    "test:compatibility:debug": "mkdir -p build/reports && cross-env TESTS=compatibility grunt compatibility",
```
Comment (gethyn1) [inline package.json]: These commands look identical

Comment (YetAnotherMatt) [reply package.json]: We can probably drop the debug test

**Comment by YetAnotherMatt** [comment]
I was really hoping we could use npm's parallel capabilities and avoid using grunt at all.
I made a brief (exploratory) start at this last weekend, but there's quite a bit of work still to do so I didn't push it to a branch. I guess we might have to bring grunt in for now :(

File: package.json (hunk 1 above)
Comment (dwybourn) [reply package.json]: @YetAnotherMatt We've removed test:compatibility:debug as this didn't work on master for myself or Mike

**Pull request merged**